Added errorSelector field for custom errors (#12653)
ethereum/solidity: Changelog.md

Language Features:

Compiler Features:

> * General: Added selector field in the Error and Event Description.

Reviewer: ```suggestion
* JSON-AST: Added selector field for errors and events.
```

Bugfixes:

> Solc-Js:

Reviewer: Please remove.

Language Features:

> 

Reviewer: Please keep the double empty lines.
